components/UserObserver.php

```diff
@@ -17,46 +17,53 @@
  * @license http://www.gnu.org/licenses/gpl-3.0.html The GNU General Public License V3.0
  */
 
-class UserObserver
+//namespace OEModule\CSDClient\components;
+
+use OEModule\mehstaffdb\components\CSDClient\CSDClient;
+
+class UserObserver extends \BaseAPI
 {
-	public function updateUser($params)
-	{
-		if (in_array($params['username'],Yii::app()->params['local_users'])) {
+
+	/**
+     * @return CSDClient
+     */
+	protected function getCSDClient(): CSDClient
+    {
+        return CSDClient::get();
+    }
+
+
+	/**
+     * Updates user from CSD database
+     *
+     * @param array $params
+	 * @return User
+     */
+	public function updateUser($params) :User
+	{		
+		/*if (in_array($params['username'],Yii::app()->params['local_users'])) {
 			return;
-		}
+		}*/
+
+		//if (Yii::app()->params['mehstaffdb_always_refresh'] || $this->isStale($params['username'])) {
+			//try {
+				$username = "WILLIAMSS";//$params['username'];
```

Comment:
is this just a placeholder for later dev work?

Reply:
Changed for `$username = $params['username'];`